src/main/scala/com/gu/zuora/creditor/CreditTransferService.scala

```diff
@@ -47,16 +47,23 @@ class CreditTransferService(
   import CreditTransferService._
   import ModelReaders._
 
-  def processExportFile(exportId: ExportId): Int = {
+  def processExportFile(exportId: ExportId): AdjustmentsReport = {
     val maybeExportCSV = downloadGeneratedExportFile(exportId)
+    maybeExportCSV.foreach{ rawCSV =>
+      logger.info("==========================================")
+      logger.info(s"csv export: $rawCSV")
+      logger.info("==========================================")
+    }
     val invoicesWhichNeedCrediting = maybeExportCSV.map(x => invoicesFromReport(ExportFile(x))).getOrElse {
       logger.error("Unable to download export of negative invoices to credit")
       Set.empty[NegativeInvoiceToTransfer]
     }
     if (invoicesWhichNeedCrediting.isEmpty) {
       logger.warn("No negative invoices reqire crediting today")
     }
-    makeCreditAdjustments(invoicesWhichNeedCrediting).size
+    val negativeInvoicesWithAutHolidayCredit = invoicesWhichNeedCrediting.count(_.ratePlanName.toLowerCase.contains("automated"))
```

Comment:
variable name typo

Reply:
also should probably be `InvoiceItem` in the name rather than just `Invoice` because the rows of the CSV are invoice line items (if you also wanted to expose the total number of actual invoices affected [a lower number] then you could do something like `.groupBy(_.invoiceNumber).size`

Reply:
i picked: `.groupBy(_.invoiceNumber).size`